### PR TITLE
Fix auto-login to new layout

### PR DESCRIPTION
**Pull request by LuisDuarte1**
This also improves the logic and reliability of the auto-login.

### CODE REVIEW

**Review by toni-santos — APPROVED** [review]

File: content-scripts/src/index.js [hunk 1]
```diff
@@ -62,6 +61,8 @@ const init = async () => {
   changeCourseCards();
   fixPagination();
   changeLayout();
+  
+  rememberLogin(data);
```
Comment (thePeras) [inline content-scripts/src/index.js:65]: Any reason to be the last function executing?

Comment (LuisDuarte1) [reply content-scripts/src/index.js:65]: It was because of the layout

Comment (LuisDuarte1) [reply content-scripts/src/index.js:65]: Your PR should fix this

File: content-scripts/src/modules/login.js [hunk 2]
```diff
@@ -30,26 +31,40 @@ export const rememberLogin = async (data) => {
         //check if there is a error while loading page
         const htmlRes = document.createElement('html');
         htmlRes.innerHTML = await res.text();
```
Comment (ttoino) [inline content-scripts/src/modules/login.js:33]: Maybe we should use the `fetchSigarraPage` function here, I remember there being some hiccups with encodings when I did it like this

File: content-scripts/src/modules/login.js [hunk 3]
```diff
@@ -58,16 +73,31 @@ function loginButtonHandler(event) {
                 console.log("Something went wrong while logging in...");
                 return;
             }
+            const htmlRes = document.createElement('html');
+            htmlRes.innerHTML = await res.text();
```
Comment (ttoino) [inline content-scripts/src/modules/login.js:77]: Same as above

File: content-scripts/src/modules/login.js [hunk 4]
```diff
@@ -1,73 +1,91 @@
...
 
 export const rememberLogin = async (data) => {
     if (!(data?.autoLogin === "on")) return;
 
+    const isAuthenticated = document.querySelector('#se-auth-form') == null;
+
     auto_login = await getStorage("auto_login");
-    console.log(auto_login);
-    if (auto_login === undefined) {
+    if (auto_login === null) {
```
Comment (thePeras) [inline content-scripts/src/modules/login.js:14]: auto_login is being `undefined` for me. Change the conditional or add `?? null` next to getStorage

**Review by thePeras — CHANGES_REQUESTED** [review]
See my comment pls

File: content-scripts/src/modules/login.js (hunk 4 above)
Comment (ttoino) [reply content-scripts/src/modules/login.js:14]: I'd just do `if (!auto_login)`, this would handle both cases

**Review by ttoino — APPROVED** [review]

**Review by thePeras — APPROVED** [review]

**Pull request merged**